paasta_tools/cli/cmds/logs.py

```diff
@@ -328,16 +372,37 @@ def get_log_reader():
 
 
 class LogReader(object):
-    def __init__(self, **kwargs):
-        pass
+    def __init__(self):
+        # Tailing, i.e actively viewing logs as they come in
+        self.SUPPORTS_TAILING = False
```

Comment:
Isn't this a constant? A logreader's supported retrieval modes won't exactly change during runtime